src/modules/git_commit.rs

```diff
@@ -40,6 +40,39 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         )),
     );
 
+    if !config.tag_disabled {
+        // Let's get repo tags names
+        let tag_names = git_repo.tag_names(None).unwrap();
+
+        let tag_and_refs = tag_names.iter().flatten().flat_map(|name| {
```

Comment:
Do we need `flatten()` followed by `flat_map()`? `flat_map()` is a semantic equivalent to `map(f).flatten()`.